packages/connectors-web/src/alchemyAuth.ts

```diff
@@ -67,25 +69,29 @@ export function alchemyAuth(parameters: AlchemyAuthOptions = {}) {
       // Connection is handled through the auth flow (sendEmailOtp -> submitOtpCode)
       // This method is called by wagmi after authentication completes
       if (!signerInstance) {
-        // TODO(v5): Update error message to reflect different auth methods available.
-        throw new Error(
-          "No signer available. Please authenticate first using sendEmailOtp and submitOtpCode.",
+        const error = new Error(
+          "No signer available. Please authenticate first using sendEmailOtp and submitOtpCode, or loginWithOauth.",
```

Comment:
can we change this error message to use the term "auth session" in place of "signer" to be consistent with the new naming conventions?